### PR TITLE
Added handy CLI arg to assume role

### PR DESCRIPTION
**Pull request by lgtml**
Although this is possible with the environment variables that would require the ~/.aws/(config|credentials) to be previously setup AND NOT have the aws creds in environment variables.

Reference: https://github.com/boto/botocore/blob/cb8d4b87fbe13b42653017b74c9a5f4097ae740c/botocore/credentials.py#L87

### CODE REVIEW

File: package/rdspgbadger.py [hunk 1]
```diff
@@ -135,7 +151,7 @@ def main():
     logger.debug("pgbadger found")
 
     try:
-        get_all_logs(args.instance, args.output, args.date, args.region)
+        get_all_logs(args.instance, args.output, args.date, args.region, args.assume_role)
```
[the remark this answers is not in the document]
Comment (lgtml) [reply package/rdspgbadger.py]: What is wrong with me :(

File: package/rdspgbadger.py [hunk 2]
```diff
@@ -71,11 +72,25 @@ def define_logger(verbose=False):
     logger.addHandler(consoleHandler)
 
 
-def get_all_logs(dbinstance_id, output, date=None, region=None):
+def get_all_logs(dbinstance_id, output, date=None, region=None, assume_role=None):
```
[the remark this answers is not in the document]
Comment (lgtml) [reply package/rdspgbadger.py]: What is wrong with me :(

**Comment by fpietka** [comment]
Nice catch!
Would it be possible for you to update the README about this new option?
If I'm not mistaken, the shape of the the `assume_role` should be `arn:aws:iam::account-of-role-to-assume:role/name-of-role` ?

**Comment by lgtml** [comment]
Correct! and I would be happy to sorry I did not think of that ahead of time

**Pull request merged**